docker/smqtk_services_2/custom-entry-point.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
I had considered this option, and the main reason I avoided it was because the scripts that initialize the database are already in the SMQTK docker image.  Adding the scripts to the postgres image would require duplicating the files in `etc/smqtk/postgress/...` so that they could all be mounted in a single directory.  If you think this duplication is acceptable, then I'd be happy to take a second shot at our own postgres image.